src/connect.rs

```diff
@@ -617,16 +596,6 @@ impl AsyncWrite for Conn {
         let this = self.project();
         AsyncWrite::poll_shutdown(this.inner, cx)
     }
-
-    fn poll_write_buf<B: Buf>(
```

Comment:
Can we forward the new `poll_write_vectored` and `is_write_vectored` methods to the inner type, here and on the other wrappers in this file?